content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,276 @@
+
+## Configure workloads for autoscaling {#workload-config}
+
+Normally, workloads don't have to be additionally configured to work with Node autoscaling, beyond
+just configuring the autoscaler. However, autoscalers provide the ability to change their behavior
+based on workload characteristics. The configuration options documented here are supported by all
+official autoscalers. Thanks to this, the workloads can be portable between clusters using different
+autoscalers. Additionally, the autoscalers provide their own, non-portable configuration options
+(see autoscaler-specific documentation).
+
+### Configure a workload not to be disrupted by consolidation
+
+{{< feature-state state="stable" for_k8s_version="v1.31" >}}
+
+You can use annotation `autoscaling.kubernetes.io/do-not-disrupt: true` on a Pod to prevent it being
+disrupted by consolidation. Consolidation should never remove a Node where a Pod annotated
+with `autoscaling.kubernetes.io/do-not-disrupt: true` is running.
+
+Example:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+  annotations:
+    autoscaling.kubernetes.io/do-not-disrupt: true
+  labels:
+    app: example
+spec:
+  containers:
+    - name: pause
+      image: pause
+```
+
+## Combine workload and Node autoscaling {#multi-scaling}
+
+### Horizontal workload autoscaling
+
+Node autoscaling usually works in response to Pods - it provisions new Nodes to accommodate
+unschedulable Pods, and then consolidates the Nodes once they're no longer needed.
+
+[Horizontal workload autoscaling](/docs/concepts/workloads/autoscaling#scaling-workloads-horizontally)
+automatically scales the number of workload replicas to maintain a desired average resource
+utilization across the replicas. In other words, it automatically creates newPods in response to
+application load, and then removes the Pods once the load decreases.
+
+You can use Node autoscaling together with horizontal workload autoscaling to autoscale the Nodes in
+your cluster based on the average real resource utilization of your Pods.
+
+If the application load increases, the average utilization of its Pods should also increase,
+prompting workload autoscaling to create new Pods. Node autoscaling should then provision new Nodes
+to accommodate the new Pods.
+
+Once the application load decreases, workload autoscaling should remove unnecessary Pods. Node
+autoscaling should in turn consolidate the Nodes that are no longer needed.
+
+If configured correctly, this pattern ensures that your application always has the VM capacity to
+handle load spikes if needed, but you don't have to pay for the capacity when it's not needed.
+
+## Related components
+
+This section describes components providing functionality related to Node autoscaling.
+
+### Descheduler
+
+The [descheduler](https://github.com/kubernetes-sigs/descheduler) is a component providing just Node
+consolidation functionality based on custom policies.
```

Comment:
The descheduler supports more than just node consolidation. For example, it will delete Pods that are restarting too many times. The relevance to node autoscaling is usually to support consolidation; `RemovePodsViolatingTopologySpreadConstraint` is also useful because it might allow recovery from a zone failure, by triggering Pod replacement and indirectly node provisioning once nodes can run in that zone.